bin/picca_Pk1D.py

```diff
@@ -103,11 +103,19 @@ def process_all_files(index_file_args):
         first_pixel_index = (np.argmax(selected_pixels) if
                              np.any(selected_pixels) else len(selected_pixels))
 
+
+        if linear_binning:
+            max_num_pixels_forest_theoretical=(1180-1050)*(delta.z_qso+1)/pixel_step  #currently no min/max restframe values defined, so hard coding for the moment
+        else:
+            max_num_pixels_forest_theoretical=(np.log(1180)-np.log10(1050))/pixel_step
         # minimum number of pixel in forest
-        min_num_pixels = args.nb_pixel_min
+        if args.nb_pixel_min is not None:
+            min_num_pixels = args.nb_pixel_min
+        else:
+            min_num_pixels = int(args.nb_pixel_frac_min*max_num_pixels_forest_theoretical)     #this is currently just hardcoding values so that spectra have a minimum length changing with z; but might be problematic for SBs
```

Comment:
Is it because the rest frame wavelength are different for SB ? In that case we should really get the rest frame limits from the deltas